lib/core/elastic_connector_actions.rb

```diff
@@ -157,12 +157,18 @@ def update_connector_sync_now(connector_id, sync_now)
         )
       end
 
-      def update_connector_last_sync_status(connector_id, last_sync_status)
+      def update_connector_sync_start(connector_id)
```

Comment:
Note: changing the name of this method will break Crawler if/when we update the utility gem (specifically [this line](https://github.com/elastic/ent-search/blob/ab343921d317d4c8857f392ab712108dd7ef41af/shared_togo/lib/shared_togo/crawler2/scheduler.rb#L60)). Once this is merged we should make a PR in ent-search to make sure it doesn't surprise anyone later.

Reply:
Good catch. Will do.